crates/air-lib/air-parser/src/ast/tests/instruction_arguments.rs

```diff
@@ -0,0 +1,27 @@
+use crate::ast::Value;
+
+use air_lambda_ast::ValueAccessor;
+/*
+ * Copyright 2022 Fluence Labs Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+use non_empty_vec::NonEmpty;
+
+#[test]
+fn bug_263_last_error_lambda() {
```

Comment:
We use issue instead of bug for such names, and please add a link to the issue in github followign this example https://github.com/fluencelabs/aquavm/blob/master/air/tests/test_module/issues/issue_137.rs#L20